roseau/load_flow/_solvers.py

```diff
@@ -225,3 +228,41 @@ def update_params(self, params: JsonDict) -> None:
 
     def params(self) -> JsonDict:
         return {"m1": self.m1, "m2": self.m2}
+
+
+class BackwardForward(AbstractSolver):
+    """A backward-forward implementation, less stable than Newton-Raphson based algorithms,
+    but can be more performant in some cases.
+    """
+
+    name = "backward_forward"
+
+    def __init__(self, network: "ElectricalNetwork") -> None:
+        """Backward-Forward constructor.
+
+        Args:
+            network:
+                The electrical network for which the load flow needs to be solved.
+        """
+        super().__init__(network=network)
+        self._cy_solver = CyBackwardForward(network=network._cy_electrical_network)
+
+    def update_network(self, network: "ElectricalNetwork") -> None:
+        self._cy_solver = CyBackwardForward(network=network._cy_electrical_network)
+
+    def update_params(self, params: JsonDict) -> None:
+        pass
```

Comment:
I would have raised an error or emitting a warning to ensure the user understands that something is weird about his code